Cargo.toml

```diff
@@ -48,15 +48,12 @@ parking_lot = ["dep:parking_lot_core"]
 # Uses `critical-section` to implement `sync` and `race` modules. in
 # `#![no_std]` mode. Please read `critical-section` docs carefully
 # before enabling this feature.
-critical-section = ["dep:critical-section", "dep:atomic-polyfill" ]
+critical-section = ["dep:critical-section", "portable-atomic" ]
 
 # Enables semver-exempt APIs of this crate.
 # At the moment, this feature is unused.
 unstable = []
 
-# Only for backwards compatibility.
-atomic-polyfill = ["critical-section"]
```

Comment:
Let’s keep this, it’s not a burden to support a feature alias, and this might prevent someone’s build from breaking